src/api/goods.ts

```
@@ -5,15 +5,32 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        return Promise.reject(
+          new Error(response.statusText),
+        );
```

Or you could just throw an error here